.github/problem-matchers/generic.json

```diff
@@ -0,0 +1,14 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "generic",
+            "pattern": [
+                {
+                    "regexp": "^([wW]arning|[wW]arn|[eE]rror|[bB]ug|WARNING|WARN|ERROR|BUG)(\\[(.*)\\])?: (.*)$",
```

Comment:
Which kind of lines are you matching with this? (please see my comment about potential candidates in https://github.com/Rust-for-Linux/linux/issues/332).

Ideally, we would have a comment with an example so that we know what the regexp is trying to match. See e.g. how it is done in https://github.com/linuxppc/linux-ci/commit/5b78114f620cbe24ede6846e764c3f5e4e7924f3

Reply:
This would match any line starting with the words: error, bug, warning, war. Not matter the case. 
 It is a very basic matching pattern.

Reply:
I see your point. Thanks for the clarification. 

 I actually don't know the exact format of the message from different error sources, 
that is why I settled to start with a basic generic pattern to catch all the different issues. 
And maybe later it would be possible to fine-tune the problem matchers. 

 Looking a bit deeper into the different problem sources:

- For QEMU and rust, the proposed example: [matcher.json](https://github.com/actions-rs/cargo/blob/master/.matcher) may be used, as it seems those issues start with: warning|warn|error. But I'm not sure all QEMU issues begin with this pattern.

  Having a generic catcher is used in some repos, and others don't use it. But it seems a sensible idea, if the format is not exactly known.
  
- For kernel oops issues, its possible to use: [linux-kernel-oopses.json](https://github.com/ClangBuiltLinux/continuous-integration2/blob/main/.github/problem-matchers/linux-kernel-oopses.json)

- For compiler-non-sorce issues: [compiler-non-sources.json](https://github.com/linuxppc/linux-ci/blob/merge/.github/problem-matchers/compiler-non-source.json) which is also used in other repo here:  https://github.com/ClangBuiltLinux/continuous-integration2/blob/main/.github/problem-matchers/compiler-non-source.json
 
- For compiler-source issues, this seems appropiate: [compiler-sources.json]( https://github.com/ClangBuiltLinux/continuous-integration2/blob/main/.github/problem-matchers/compiler-source.json)

 I have created #768 to see if this works fine, and have a more detailed solution.

Reply:
Yes, sure. We can use the other PR #768